modules/server/src/network/setup/query.ts

```diff
@@ -22,15 +22,23 @@ type NetworkQueryResult = PromiseFulfilledResult<{
  *
  * @throws Unexpected data error
  */
-const fetchRemoteSchema = async (
+const fetchNodeAggregations = async (
 	config: NetworkConfig,
 ): Promise<GQLTypeQueryResponse | undefined> => {
 	const { graphqlUrl, documentType } = config;
+	/*
+	 * documentType is an entire field name / type name in the case of a root field
+	 * eg. `file:file`, `torontoFile:torontoFile`
+	 * it also prefixes dynamic type names like *Connection and *Aggregations
+	 * eg. `aggregations: fileAggregations`, `hits: fileConnection`
+	 */
 	try {
+		// targeting just the "aggregations" field using type name
+		const typename = `${documentType}Aggregations`;
 		const response = await fetchGql({
 			url: graphqlUrl,
 			gqlQuery: gqlAggregationTypeQuery,
-			variables: { documentName: documentType },
+			variables: { documentTypeName: typename },
```

Comment:
Camel case `typeName` ?

Reply:
consistent with usage in relation to gql schema. although this is a function not directly in the resolvers, it makes sense to keep it consistent case wise as it is referring to the same entity eg. typename is referring to __typename not __typeName

Reply:
Got it 👍